monitoring-plugins.spec.in

```diff
@@ -95,107 +30,117 @@ BuildRequires:	postgresql-devel
 BuildRequires:	procps
 BuildRequires:	samba-client
 BuildRequires:	/usr/bin/mailq
-%endif
 
+# Don't let rpmbuild try to figure out the dependancies.
+# It's far too confusing for it.
+AutoReqProv: no
```

Comment:
i'm sure this is very outdated nowadays.

the only reason for `AutoReqProv: no` is that you do not want to pull dependencies, but handle them yourself. this is not recommended, but then again allows installing the package so that postgresql packages are not pulled (and pgsql plugin will fail to find libraries)